docs/cli_gettingstarted.md

```diff
@@ -97,6 +73,30 @@ You can import the corresponding private key to a wallet solution like MetaMask.
 
 >The keys are locally stored and EthVigil does not have access to them. You can check the `settings.json` file once init is complete.
 
+### ✅ Reuse your in-browser local wallet settings (No Metamask involved)
```

Comment:
Add a line above skip to next step.

Reply:
Added in commit 263fbad

Review please.

Reply:
c6cc417